packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebSettingsHostApiImpl.java

```diff
@@ -132,4 +132,11 @@ public void setTextZoom(@NonNull Long instanceId, @NonNull Long textZoom) {
     final WebSettings webSettings = Objects.requireNonNull(instanceManager.getInstance(instanceId));
     webSettings.setTextZoom(textZoom.intValue());
   }
+
+  @Nullable
```

Comment:
Does the underlying API really return a nullable string?

Reply:
I tested this out and it looks like it is always nonnull. Updated.